### PR TITLE
site: move the remainder of /docs to /site

### PR DESCRIPTION
**Pull request by davecheney**
Updates #1619

Signed-off-by: Dave Cheney <dave@cheney.net>

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,17 +1,10 @@
...
-  * [TLS support](tls.md)
-  * [HTTPProxy API](httpproxy.md)
-  * [IngressRoute API](ingressroute.md)
-* [About Contour and Envoy](about.md)
-* [Image tagging policy](tagging.md)
-* [Architecture](architecture.md)
-* [Supported Annotations](annotations.md)
-* [Examples directory](../examples/README.md)
+* Guides and How-to's have moved to [projectcontour.io/guides](https://projectcontour.io/guides)
+* Versioned release documetaion has moved to [projectcontour.io/docs](https://projectcontour.io/docs)
```
Comment (jpeach) [inline docs/README.md]: spelling "documentation"

File: docs/README.md [hunk 2]
```diff
@@ -1,17 +1,10 @@
...
+* Project related and non versioned documetation has moved to [projectcontour.io/resources](https://projectcontour.io/resources/)
```
Comment (jpeach) [inline docs/README.md]: "non-versioned"

File: site/_docs_1_0/annotations.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+---
 
-Annotations are used in Ingress Controllers to configure features that are not
-covered by the Kubernetes Ingress API.
+Annotations are used in Ingress Controllers to configure features that are not covered by the Kubernetes Ingress API.
 
-Some of the features that have been historically configured via annotations are
-supported as first-class features in Contour's [IngressRoute
-API](ingressroute.md), which provides a more robust configuration interface over
+Some of the features that have been historically configured via annotations are supported as first-class features in Contour's [IngressRoute API](ingressroute.md), which provides a more robust configuration interface over
```
Comment (jpeach) [inline site/_docs_1_0/annotations.md]: Does this link work? I expect it should be `{% link Ingressroute.md %}` or something?

File: site/_resources/ingressroute-to-httpproxy.md [hunk 4]
```diff
@@ -138,7 +138,7 @@ spec:
 
 HTTPProxy offers additional ways to match incoming requests to routes.
 This document covers the conversion between the routing offered in IngressRoute and HTTPProxy.
-For a broader discussion of HTTPProxy routing, see the [Routing section of the HTTPProxy documentation]({{ site.github.repository_url }}/tree/master/docs/httpproxy.md#Routing).
+For a broader discussion of HTTPProxy routing, see the [Routing section of the HTTPProxy documentation]({% link _docs_1_0/httpproxy.md %}#Routing).
```
Comment (jpeach) [inline site/_resources/ingressroute-to-httpproxy.md]: I think the anchors are all lowercased.

File: site/_docs_1_0/annotations.md (hunk 3 above)
Comment (davecheney) [reply site/_docs_1_0/annotations.md]: link does work, but it only seems to take absolute references. fixed.

File: site/_resources/ingressroute-to-httpproxy.md (hunk 4 above)
Comment (davecheney) [reply site/_resources/ingressroute-to-httpproxy.md]: fixed

**Pull request merged**